readme.md

```diff
@@ -86,19 +86,35 @@ filter(is(Number), [0, '1', 2, null]); // [0, 2]
 
 ## Partial Application
 
-The process of getting a function with lesser arity compared to the original
-function by fixing the number of arguments is known as partial application.
+Partially applying a function means creating a new function by pre-filling some of the arguments to the original function.
+
 
 ```js
-let sum = (a, b) => a + b;
+// Helper to create partially applied functions
+// Takes a function and some arguments
+let partial = (f, ...args) =>
```

Comment:
.bind is more terse than defining an implementation but I think it hides the important information about how partial application works and confuses by requiring an unnecessary context.